extensions/resource-deployment/src/interfaces.ts

```diff
@@ -296,7 +305,7 @@ export interface FieldInfo extends SubFieldInfo, FieldInfoBase {
 	fontWeight?: FontWeight;
 	links?: azdata.LinkArea[];
 	editable?: boolean; // for editable drop-down,
-	enabled?: boolean;
+	enabled?: boolean | DynamicEnablementInfo;
```

Comment:
interesting, I didn't know we have the enabled field, not sure what is always disabled field used for.

Reply:
https://github.com/microsoft/azuredatastudio/pull/9036

For fields that we want to be styled like the rest of the input components but not be able to be modified apparently - although honestly I don't recall what fields these were and nothing seems to use it. I figured leaving it in doesn't hurt in case someone did want to for some reason.